.travis.yml

```diff
@@ -43,8 +43,8 @@ before_deploy:
 deploy:
   provider: releases
   api_key:
-    #sem-geologist:
-    secure: n1CcuRSDf9GxJBFYr5GXGAhYL+sY1If1IzgMVY0YeeS4zHvIgn4DWf32iE+k/7GN6EsS59scVagg5zbDsO+DxVvYFT4BkqvBKz7SoHo1TMX7oqCHlwi+RnnUUBqeBCuq8zC7x9IPtfoLPXoFeUiGeYoAWcB+iPKrUIoWkBdAGi0fEl7b8K8JAn9kVF5xoqe4V4Bwcxnd2fTD2eI7JeA+mcRabGO53ERfZiJZqx9Do7YBKSG31E6IVcUCoehc2VFvTKwA7yzEXvPWyrmrKlRdYWjyLHRk+wDflHWkvzXGcr/6hewgxI5KjJ6eERWKZ9PqxukGr5G4lixMUlfgtNSxbnqyvBN7L0lGwpMBpT9XIjwaikcHd0bUgP6W8hNFP9OFVcrmMdYil7ko5OIZTw3Dc27RH6mLr/653hgvCDPg+GO/wfx0mVGAvn+vl7u0i/kfH0sMjvBKWmy3CEyt3jbrrawXwZa1wPgcCEwt/7f3WgiMEOOiZMrVQbB1QtLOXFi6KXr0gCKS3mShr3cmr4r6p6Xi95zNLQ0IM/B+mC3+/a8g1fDFYOJjzxlbQjn9WEV5Bzj8iw/VLx1ndnE2BE1A7bvk+yWKb1WiGP8n60OudQpHU5yrtwWCbrjUCl1PoYvB1zrZClAAFTr7/oxQRRACZ80ipYhSOSGC0FWPWSJ/gVs=
+    #francisco-dlp:
+    secure: dTt5L3HsIrfTOvnU8QFqwMu+Ghpuzp+vr0y7skRbqdFBCmPfjbAe2l24haMidVC1x0ZikHGgv9krEd/aKEBLi6QM+E2KsxiXUJWlhTFMH5o8qsMTBSAS2ChfQ32Jlktg8oK0FMs+UO6EFbMz1cr7m1Fgc0Ikds1ZCzev6Uke2Zk=
```

Comment:
You are right, as far as I understood, travis cli generates the token using
GH api and encrypts it.

2016-07-13 19:37 GMT+01:00 Petras Jokubauskas <notifications@github.com>:

> In .travis.yml
> <https://github.com/hyperspy/hyperspy/pull/1140#discussion_r70683587>:
>
> > @@ -43,8 +43,8 @@ before_deploy:
> >  deploy:
> >    provider: releases
> >    api_key:
> > -    #sem-geologist:
> > -    secure: n1CcuRSDf9GxJBFYr5GXGAhYL+sY1If1IzgMVY0YeeS4zHvIgn4DWf32iE+k/7GN6EsS59scVagg5zbDsO+DxVvYFT4BkqvBKz7SoHo1TMX7oqCHlwi+RnnUUBqeBCuq8zC7x9IPtfoLPXoFeUiGeYoAWcB+iPKrUIoWkBdAGi0fEl7b8K8JAn9kVF5xoqe4V4Bwcxnd2fTD2eI7JeA+mcRabGO53ERfZiJZqx9Do7YBKSG31E6IVcUCoehc2VFvTKwA7yzEXvPWyrmrKlRdYWjyLHRk+wDflHWkvzXGcr/6hewgxI5KjJ6eERWKZ9PqxukGr5G4lixMUlfgtNSxbnqyvBN7L0lGwpMBpT9XIjwaikcHd0bUgP6W8hNFP9OFVcrmMdYil7ko5OIZTw3Dc27RH6mLr/653hgvCDPg+GO/wfx0mVGAvn+vl7u0i/kfH0sMjvBKWmy3CEyt3jbrrawXwZa1wPgcCEwt/7f3WgiMEOOiZMrVQbB1QtLOXFi6KXr0gCKS3mShr3cmr4r6p6Xi95zNLQ0IM/B+mC3+/a8g1fDFYOJjzxlbQjn9WEV5Bzj8iw/VLx1ndnE2BE1A7bvk+yWKb1WiGP8n60OudQpHU5yrtwWCbrjUCl1PoYvB1zrZClAAFTr7/oxQRRACZ80ipYhSOSGC0FWPWSJ/gVs=
> > +    #francisco-dlp:
> > +    secure: dTt5L3HsIrfTOvnU8QFqwMu+Ghpuzp+vr0y7skRbqdFBCmPfjbAe2l24haMidVC1x0ZikHGgv9krEd/aKEBLi6QM+E2KsxiXUJWlhTFMH5o8qsMTBSAS2ChfQ32Jlktg8oK0FMs+UO6EFbMz1cr7m1Fgc0Ikds1ZCzev6Uke2Zk=
>
> but if it will work, maybe there is not only one way. the way I described
> is checked and 100% works.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hyperspy/hyperspy/pull/1140/files/526509e60e7766dd8ab3ee80509a5f54ff54e585..99bf866707924169edaaff119da93d2028503f8b#r70683587>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA8aF-W4tPdrEb7XH_x7vxEatAtNY2hnks5qVTB4gaJpZM4JLo-f>
> .
>